client/src/pages/ListPaymentOptions.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import CreditCardIcon from '@material-ui/icons/CreditCard';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import AccountBalanceIcon from '@material-ui/icons/AccountBalance';
+import AttachMoneyIcon from '@material-ui/icons/AttachMoney';
+
+function ListPaymentOptions(props) {
```

Comment:
```suggestion
function ListPaymentOptions({setStripePage}) {
```

Reply:
you can pass your `setStripePage` this way without reaching into the `prop` object every time